source/fe/fe_tools.cc

```diff
@@ -2519,7 +2519,12 @@ namespace FETools
   }
 
 
-
+  template <int dim>
+  FiniteElement<dim> *
+  get_fe_from_name (const std::string &parameter_name)
+  {
+    return get_fe_by_name<dim,dim> (parameter_name);
+  }
 
 
```

Comment:
We usually put three empty lines between functions. Could you adjust it above and below this function? (I know that we are not consistent in the whole `fe_tools.cc`.)